### ISSUE TITLE
Some editor buttons of the Cart and Checkout blocks inherit the theme styling when they shouldn't

### ISSUE THREAD

**Issue by Aljullu** [woocommerce/woocommerce-gutenberg-products-block#8571]
## To reproduce

0. Make sure you don't have any payment method set up.
1. Install TT3 and go to Appearance > Editor > edit one of the templates > Styles (white and black circle on the top right) > Browse Styles and select the _Whisper_ style.
2. Create a post or page and add the Cart and Checkout blocks.
3. Notice some of the buttons inherit the theme styling (text color, borders, etc.), making them difficult to read:
<img src="https://user-images.githubusercontent.com/3616980/221821518-f819d6d9-33c4-40e8-8e8d-7d6af45f2362.png" alt="" width="493" />

## Expected behavior

Buttons displayed in this kind of notices shouldn't inherit the theme styling.

**Comment by ralucaStan** [comment]
Thank you @Aljullu !

**Comment by nielslange** [comment]
Internal discussion: p1679990642217489-slack-C8X6Q7XQU

**Comment by tarunvijwani** [comment]
"Configure payments methods" is a link styled to be displayed as a button. 

We have `Setup a Terms and Conditions page` and `Setup a Privacy Policy page` buttons for the internal actions on the checkout block page. However, they're not styled like a blue wp-admin button. 

The goal is to have them as links to be styled as a blue wp-admin button that doesn't inherit any theme style:
- Configure payment method
- Setup a Terms and Conditions page
- Setup a Privacy Policy page

We will now convert the `Configure payment method` link to a button. And later, we will convert all the above to a link styled as a button. Ticket for it: https://github.com/woocommerce/woocommerce-blocks/issues/8944

I'm adding this ticket to Global styles Epic to convert the `Configure payment method` link to a button, as it will need a separate PR.

**Comment by mikejolley** [comment]
> We will now convert the Configure payment method link to a button.

I don't think we should mess with the semantics here; it is a link. What we can do is make it match the links in the inspector, where we have a small arrow icon beside a link to another admin page.
